functions.php

```diff
@@ -347,6 +354,70 @@ function render_related_posts_block(array $attributes): string
     return do_blocks($output);
 }
 
+/**
+ * Custom block render function for Bottom page navigation
+ *
+ * @param array  $attributes Array of dynamic attributes to render section.
+ *
+ * @return string HTML markup for front end.
+ */
+// phpcs:ignore SlevomatCodingStandard.Functions.UnusedParameter.UnusedParameter
+function render_navigation_block(array $attributes): string
+{
+    global $post;
+
+    $menu_items = wp_get_nav_menu_items('Navigation Bar Menu');
```

Comment:
I'm wondering if this might cause issues because this string can be translated 🤔 I see that for the navigation bar [this is what we use](https://github.com/greenpeace/planet4-master-theme/blob/main/src/MasterSite.php#L654-L655) to get the items